src/hdmf/term_set.py

```diff
@@ -0,0 +1,91 @@
+from collections import namedtuple
+
+
```

Comment:
Not critical for this PR, but it would be nice to add `_repr_html_` function similar to #883 for TermSet as well to be able to render term sets nicely in the docs and Jupyter notebooks. Similarly, it would be nice to create a `__repr__` to make a nice plain-text rendering for TermSets.